src/main/resources/db/migration/0300/V320__AdHocPlotObservations.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE tracking.ad_hoc_observation_types (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE tracking.ad_hoc_plot_observations (
+    ad_hoc_observation_type_id INTEGER NOT NULL REFERENCES tracking.ad_hoc_observation_types,
+    boundary GEOMETRY(MultiPolygon) NOT NULL,
```

Comment:
@sgrimm since this is an auto-sized plot, created by the SW corner, do you have a preference if that is calculated server side or client side? I believe @joshbuchea is planning on doing it client side but I want to make sure that is not going to conflict with the (possibly) standardized way we are doing this already.

Reply:
Does anyone know the correct method to calculate a plot boundary starting from the southwest corner? 

Here’s my current (likely incorrect) attempt:

```ts
export const getPlotBoundaryPolygon = (swCorner: Position) => {
  const southwest = point(swCorner);
  const northwest = destination(southwest, 30, 0, { units: 'meters' }); // 30m north
  const northeast = destination(northwest, 30, 90, { units: 'meters' }); // 30m east
  const southeast = destination(southwest, 30, 90, { units: 'meters' }); // 30m east

  const coordinates = [
    southwest.geometry.coordinates,
    northwest.geometry.coordinates,
    northeast.geometry.coordinates,
    southeast.geometry.coordinates,
    southwest.geometry.coordinates,
  ];

  return turfPolygon([coordinates]);
};
```

Also, without providing the NW corner explicitly, wouldn’t multiple boundaries be possible from the same SW corner depending on how other points are determined?